src/Core/Grid/Definition/GridDefinitionInterface.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Grid\Definition;
+
+use PrestaShop\PrestaShop\Core\Grid\Action\BulkAction;
+use PrestaShop\PrestaShop\Core\Grid\Action\BulkActionCollectionInterface;
+use PrestaShop\PrestaShop\Core\Grid\Action\GridActionCollectionInterface;
+use PrestaShop\PrestaShop\Core\Grid\Action\GridActionInterface;
+use PrestaShop\PrestaShop\Core\Grid\Action\RowActionCollectionInterface;
+use PrestaShop\PrestaShop\Core\Grid\Column\Column;
+use PrestaShop\PrestaShop\Core\Grid\Action\RowAction;
+use PrestaShop\PrestaShop\Core\Grid\Column\ColumnCollectionInterface;
+use PrestaShop\PrestaShop\Core\Grid\Column\ColumnInterface;
+
+/**
+ * Interface GridDefinitionInterface defines contract for grid definition
+ */
+interface GridDefinitionInterface
+{
+    /**
+     * Get grid name
+     *
+     * @return string
+     */
+    public function getName();
+
+    /**
+     * Get unique grid indentifier
+     *
+     * @return string
+     */
+    public function getIdentifier();
+
+    /**
+     * Get grid columns
+     *
+     * @return ColumnCollectionInterface|ColumnInterface[]
```

Comment:
```diff
- ColumnCollectionInterface|ColumnInterface[]
+ ColumnCollectionInterface
```
only on type accepted please, more robust and easier to unit test (I'll write all the tests before 1.7.5.x release)